tests/Makefile.simulation-test

```diff
@@ -35,7 +35,7 @@ BASESEED ?= 1
 # The number of runs (with different random seeds)
 RUNCOUNT ?= 1
 
-CONTIKI=../..
+CONTIKI ?= ../..
```

Comment:
Is this change necessary?

Reply:
Yes, file `examples/libs/unit-tests/Makefile` is on other deep in files tree than this default path expects. It cannot be compiled without this change.